jsonParser.go

```diff
@@ -130,39 +128,135 @@ type Link struct {
 	ContentVersion      string `json:"content-version"`
 	IntendedApplication string `json:"intended-application"`
 }
-
 type Primary struct {
 	URL string `json:"URL"`
 }
 type Resource struct {
 	Primary Primary `json:"primary"`
 }
-type Issued struct {
-	DateParts [][]int `json:"date-parts"`
-}
 type JournalIssue struct {
-	Issue           *string          `json:"issue"`
-	PublishedOnline *PublishedOnline `json:"published-online"`
-	PublishedPrint  *PublishedPrint  `json:"published-print"`
+	Issue           *string    `json:"issue"`
+	PublishedOnline *DateParts `json:"published-online"`
+	PublishedPrint  *DateParts `json:"published-print"`
 }
 type IssnType struct {
 	Value string `json:"value"`
 	Type  string `json:"type"`
 }
-type PublishedOther struct {
-	DateParts [][]int `json:"date-parts"`
-}
-type Published struct {
-	DateParts [][]int `json:"date-parts"`
-}
-
 type Relation struct {
 	Cites []any `json:"cites"`
 }
-
 type License struct {
 	URL            string  `json:"URL"`
 	Start          Indexed `json:"start"`
 	DelayInDays    int     `json:"delay-in-days"`
 	ContentVersion string  `json:"content-version"`
 }
+
+//
+func (pub *CrossRef) GetSimpleTitle() []string {
```

Comment:
I think this method can be abstracted to a function. It should probably take `[]string` and return `string`.